modules/sdk-api/src/bitgoAPI.ts

```diff
@@ -772,6 +773,32 @@ export class BitGoAPI implements BitGoBase {
     return authParams;
   }
 
+  /**
+   * Validate the passkey response is in the expected format
+   * Should be as is returned from navigator.credentials.get()
+   */
+  validateWebauthnResponse(params: AuthenticateWithPasskeyOptions): void {
+    if (!_.isString(params.username)) {
+      throw new Error('expected string username');
+    }
```

Comment:
We'll actually extract the username from the webauthn response's userHandle in a follow-up. That way - UI can send data as it.